core/src/Baked/Ui/DataTable.cs

```diff
@@ -12,11 +12,21 @@ public record DataTable : IComponentSchema
     public VirtualScroller? VirtualScrollerOptions { get; set; }
     public Footer? FooterTemplate { get; set; }
     public Export? ExportOptions { get; set; }
+    public Action? ActionTemplate { get; set; }
 
-    public record Column(string Prop)
+    public record Action : Column
     {
-        public string Prop { get; set; } = Prop;
-        public Conditional Component { get; set; } = Components.Conditional();
+        public Action() : base("Action")
+        {
+            Exportable = false;
+            Frozen = false;
+        }
+    }
```

Comment:
i didn't get why we needed `Action` record. We never set `bool?` props to `false`, we keep them null to minimize descriptors. Without it this record only sets the column key to `"Action"` which can be easily made through configs and conventions.

can we remove `Action` record and use `Column` in `ActionTemplate`?